README.md

```diff
@@ -1,27 +1,35 @@
 <img src="./docs/img/openshift.png" width="180" align="right">
 
-# Odo - Developer Focused CLI for OpenShift
+# Odo - Developer focused CLI for OpenShift
 
 [![Build Status](https://travis-ci.org/redhat-developer/odo.svg?branch=master)](https://travis-ci.org/redhat-developer/odo) [![codecov](https://codecov.io/gh/redhat-developer/odo/branch/master/graph/badge.svg)](https://codecov.io/gh/redhat-developer/odo) [![CircleCI](https://circleci.com/gh/redhat-developer/odo/tree/master.svg?style=svg)](https://circleci.com/gh/redhat-developer/odo/tree/master) [![mattermost](/docs/img/mattermost.svg)](https://chat.openshift.io/developers/channels/odo)
 
-A fast iterative tool for deploying your source code straight to OpenShift.
+OpenShift Do (odo) is a  fast iterative CLI tool for developers who write, build, and deploy applications on OpenShift.
+Existing tools such as `oc` are more operations-focused and require a deep-understanding of Kubernetes and OpenShift concepts.
 
-## Features
+odo is  an opinionated CLI tool that supports fast, iterative development and that abstracts away complex Kubernetes and OpenShift concepts, thus allowing developers to focus on what's most important to them: code.
```

Comment:
We have had a discussion at the face-to-face, but when odo is used as a sentence it's lower-case. But when odo is used at the beginning, it's upper-case.

Reply:
This is a run-off sentence and has too many `and's`, needs to be reworded: `odo is  an opinionated CLI tool that supports fast, iterative development and that abstracts away complex Kubernetes and OpenShift concepts, thus allowing developers to focus on what's most important to them: code.`

`and that abstracts` sounds odd.

There's too much filler in this sentence.

Reply:
Afaiu, odo is a cli command and when used it should be with backticks. I would suggest that we consistently use `odo` everywhere we use it, or use OpenShift Do everywhere, especially in the text. We could use attributes to render oso as OpenShift Do in the published docs WDYT?